src/data_availability.rs

```diff
@@ -4,15 +4,13 @@ mod api;
 pub mod codec;
 pub mod node_state;
 
-#[cfg(test)]
+mod blocks_fjall;
 mod blocks_memory;
-mod blocks_sled;
 
-// Alias one of the two to blocks for tests
-#[cfg(test)]
-use blocks_memory::Blocks;
-#[cfg(not(test))]
-use blocks_sled::Blocks;
+// Pick one of the two implementations
+use blocks_fjall::Blocks;
+//use blocks_memory::Blocks;
```

Comment:
unused ?

Reply:
Not needed for now but I kept it around in case it's useful as a baseline / in the future